File: backend/src/test/java/org/luxons/sevenwonders/game/cards/CardBackTest.java
package org.luxons.sevenwonders.game.cards;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class CardBackTest {

    @Test
    public void initializedWithImage() throws Exception {
        String imagePath = "whateverimage.png";
        CardBack back = new CardBack(imagePath);
        assertEquals(imagePath, back.getImage());
    }
}

File: sw-engine/src/test/kotlin/org/luxons/sevenwonders/game/cards/CardBackTest.kt
package org.luxons.sevenwonders.game.cards

import org.junit.Test
import org.luxons.sevenwonders.game.api.cards.CardBack
import kotlin.test.assertEquals

class CardBackTest {

    @Test
    fun initializedWithImage() {
        val imagePath = "whateverimage.png"
        val (image) = CardBack(imagePath)
        assertEquals(imagePath, image)
    }
}
